src/components/vue-form-renderer.vue

```diff
@@ -205,42 +207,45 @@ export default {
         this.$emit('submit', this.transientData);
       }
     },
-    validateElements(elements) {
-      elements.forEach(element => {
-        if (element.validator && element.validator.errorCount !== 0) {
-          this.valid = false;
-          this.errors.push(element.validator.errors.errors);
+    getDataAndRules(items) {
+      let data = {}, rules = {};
+      items.forEach(item => {
+
+        if (Array.isArray(item)) {
+          const [data1, rules1] = this.getDataAndRules(item);
+          data = {...data, ...data1};
+          rules = {...rules, ...rules1};
         }
-      });
-    },
-    validateContainer(container) {
-      if (container.$refs && container.$refs.container) {
-        this.validateContainer(container.$refs.container);
-      }
-      container.forEach(element => {
-        if (element.$refs && element.$refs.elements) {
-          this.validateElements(element.$refs.elements);
+
+        if (item.items) {
+          const [data2, rules2] = this.getDataAndRules(item.items);
+          data = {...data, ...data2};
+          rules = {...rules, ...rules2};
+        }
+
+        if (item.config && item.config.name && item.config.validation) {
+          data[`${item.config.name}`] = this.data[item.config.name];
+          rules[`${item.config.name}`] = item.config.validation;
         }
       });
+
+      return [data, rules];
     },
     isValid() {
       this.errors = [];
-      this.valid = true;
+      const [ data, rules ] = this.getDataAndRules(this.config);
 
-      if (this.$refs && this.$refs.elements) {
-        this.validateElements(this.$refs.elements);
+      this.dataTypeValidator = new Validator( data, rules, null);
```

Comment:
Looking at the docs, it seems the 3rd argument here is an optional `customErrorMessages` object; you can leave this argument empty instead of passing `null`.